modules/apps/frontend-theme/frontend-theme-css-custom-properties-sample-web/src/main/resources/META-INF/resources/js/CssCustomPropertiesSample.js

```diff
@@ -0,0 +1,320 @@
+	'success',
+	'info',
+	'warning',
+	'danger',
+	'light',
+	'lighter',
+	'gray',
+	'dark',
+];
+const spacers = [
+	'spacer-1',
+	'spacer-2',
+	'spacer-3',
+	'spacer-4',
+	'spacer-5',
+	'spacer-6',
+	'spacer-7',
+	'spacer-8',
+	'spacer-9',
+	'spacer-10',
+];
+const borders = [
+	'border-radius',
+	'border-radius-sm',
+	'border-radius-lg',
+	'border-radius-rounded',
+	'border-radius-circle',
+	'border-radius-pill',
+];
+const shadows = ['shadow', 'shadow-sm', 'shadow-lg'];
+const ratios = [
+	'aspect-ratio',
+	'aspect-ratio-16-to-9',
+	'aspect-ratio-8-to-3',
+	'aspect-ratio-4-to-3',
+];
+const fontFamilies = [
+	'font-family-sans-serif',
+	'font-family-monospace',
+	'font-family-base',
+];
+const fontWeights = [
+	'font-weight-lighter',
+	'font-weight-light',
+	'font-weight-normal',
+	'font-weight-semi-bold',
+	'font-weight-bold',
+	'font-weight-bolder',
+];
+const headings = ['h1', 'h2', 'h3', 'h4', 'h5', 'h6'];
+const displays = ['display-1', 'display-2', 'display-3', 'display-4'];
+
+const CcpGroup = ({children, group, md, title}) => {
+	return (
+		<ClayLayout.Col
+			className={'ccp-group ccp-group-' + group}
+			md={md}
+			size={!md && '12'}
+		>
+			{title && <h2>{title}</h2>}
+			{children && <div className="ccp-items">{children}</div>}
+		</ClayLayout.Col>
+	);
+};
+
+const CcpItem = ({children, label, sample}) => {
+	return (
+		<div className="ccp-item">
+			<span className={classNames('ccp-sample', sample)}>{children}</span>
+
+			{(label || sample) && (
+				<span className="ccp-label">{label || sample}</span>
+			)}
+		</div>
+	);
+};
+
+const CssCustomPropertiesSample = () => {
+	const [fade, setFade] = useState(false);
+	const [collapse, setCollapse] = useState(false);
+
+	return (
+		<div className="ccp">
+			<ClayLayout.ContainerFluid>
+				<ClayLayout.Row>
+					<ClayLayout.Col>
+						<h1>
+							{Liferay.Language.get(
+								'css-custom-properties-sample'
+							)}
+						</h1>
+					</ClayLayout.Col>
+
+					<CcpGroup
+						group="colors"
+						title={Liferay.Language.get('grays')}
+					>
+						{grays.map((item) => (
+							<CcpItem
+								key={item}
+								label={item}
+								sample={`bg-${item}`}
+							/>
+						))}
+					</CcpGroup>
+
+					<CcpGroup
+						group="colors"
+						title={Liferay.Language.get('theme-colors')}
+					>
+						{themeColors.map((item) => (
+							<CcpItem
+								key={item}
+								label={item}
+								sample={`bg-${item}`}
+							/>
+						))}
+					</CcpGroup>
+
+					<CcpGroup
+						group="spacers"
+						md="4"
+						title={Liferay.Language.get('spacers')}
+					>
+						{spacers.map((item) => (
+							<CcpItem
+								key={item}
+								label={item}
+								sample={item.replace('spacer', 'pr')}
+							/>
+						))}
+					</CcpGroup>
+
+					<CcpGroup
+						group="borders"
+						md="4"
+						title={Liferay.Language.get('borders')}
+					>
+						{borders.map((item) => (
+							<CcpItem
+								key={item}
+								label={item}
+								sample={item.replace(
```

Comment:
It's more readable and inteligible to create a custom object for each item than doing this replace.

```
const borders = [{
   label: 'border-radius',
   sample: 'rounded'
},
{
   label: 'border-radius-sm',
   sample: 'rounded-sm'
},
{
   label: 'border-radius-lg',
   sample: 'rounded-lg'
},
{
   label: 'border-radius-rounded',
   sample: 'rounded-rounded'
},
{
   label: 'border-radius-circle',
   sample: 'rounded-circle'
},
{
   label: 'border-radius-pill',
   sample: 'rounded-pill'
},
```

```
{borders.map((item) => (
	<CcpItem
		key={item.label}
		label={item.label}
		sample={item.sample}
	/>
))}
```